build/windows/installer/conf/fluent-bit.conf

```diff
@@ -1,25 +1,61 @@
 [SERVICE]
-    Flush        15
-    Daemon       Off
-    Log_Level    info
-    Log_File      /etc/fluent-bit/fluent-bit.log
+    Flush           15
```

Comment:
This has to be configurable like in Linux i.e. ${SERVICE_FLUSH_INTERVAL}, right?